### PR TITLE
Update environment.md

### PR DESCRIPTION
**Pull request by dwbrown2**
Highlight that a Kubernetes cluster is required to run Kubecost.

Order in a way that matches typical customer question progress.

### CODE REVIEW

**Review by bstuder99 — DISMISSED** [review]

**Comment by dwbrown2** [comment]
Thanks, @bstuder99! Ready to merge unless @kwombach12 or @thomasvn have feedback. More improvements to come for this resource.

File: install-and-configure/install/environment.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+
+* Kubecost runs in any Kubernetes v1.8 (Aug 2017) or greater environment.
+* Kubernetes 1.25 is officially supported as of v1.99.0
+
+## Supported Cluster Types
+
+* ​Managed Kubernetes clusters (e.g. EKS, GKE, AKS) *most common*
+* ​Kubernetes distributions (e.g. OpenShift, DigitalOcean, Rancher, Tanzu)
+* ​Bootstrapped Kubernetes cluster​
+* On-prem and air-gapped using custom [pricing sheets](https://docs.kubecost.com/install-and-configure/install/provider-installations/air-gapped#how-do-i-configure-prices-for-my-on-premise-assets)
```
Comment (teevans) [inline install-and-configure/install/environment.md:16]: YES

**Comment by srpomeroy** [comment]
Fixed the conflict and also adjusted the wording to point to the contact us page instead of specifying the email (we are trying to get away from that)

**Comment by thomasvn** [comment]
Looks good to me! I like the new flow of the document.

**Review by bstuder99 — APPROVED** [review]

**Pull request merged**